### ISSUE TITLE
[Feedback] OData Analytics Endpoint - /ai/tools/analytics/odata-analytics-endpoint/

### ISSUE THREAD

**Issue by MaciejLitwiniec** [Cognigy/Documentation#18]
Querying part contain OData Query Language Operators with $prefix.
https://docs.cognigy.com//ai/tools/analytics/odata-analytics-endpoint/?h=odata#querying 

There are also usage examples, which are v2.2 based, which is deprecated as of 4.48.
"Removed deprecated OData endpoints versions 2.0, 2.1, and 2.2"

Given examples does not work with OData API version 2.3, like
https://<our_domain>/v2.3/Analytics?$filter=projectId eq 'xxx'&apikey=yyy

Instead, working example would be (without $)
https://<our_domain>/v2.3/Analytics?filter=projectId eq 'xxx'&apikey=yyy

**Comment by CognigyTechWriter** [comment]
Thanks for submitting this. This is now in our backlog and will be checked and improved.

**Comment by CognigyTechWriter** [comment]
We have updated our [OData Analytics Endpoint](https://docs.cognigy.com/ai/tools/analytics/odata-analytics-endpoint/) page.

Updates include examples and version information.

However, we have tested this internally and the '$' symbol is still required for version 2.3 OData requests when filtering.

Therefore this was not changed in the documentation. 

We will close this issue at this time, thanks for bringing this to our attention.
